src/camera_display.cpp

```diff
@@ -254,8 +258,20 @@ void CameraPub::preRenderTargetUpdate(const Ogre::RenderTargetEvent& evt)
 
 void CameraPub::postRenderTargetUpdate(const Ogre::RenderTargetEvent& evt)
 {
-  // TODO(lucasw) allow this to be throttled down
   // Publish the rendered window video stream
+  float framerate = -1;
+  ros::param::get("/rviz_camera_framerate", framerate); /// Absolute name as RVIZ is started anonymously and other nodes need a fixed name
```

Comment:
The parameter server is meant to be used on initialization or otherwise infrequently, so only get the parameter in onInitialize()- if the framerate needs to be changed live do it with a service callback or even a topic (an rviz plugin gui control made using a float property would be nice too).